icarus_backend/clearance/testClearanceViews.py

```diff
@@ -0,0 +1,59 @@
+from django.test import TestCase
+from django.urls import reverse
+from django.contrib.gis.geos import Polygon
+from django.utils.dateparse import parse_datetime
+from django.contrib.auth.models import User
```

Comment:
`from django.contrib.gis.geos import Polygon
from django.utils.dateparse import parse_datetime
from django.contrib.auth.models import User
` 
None of these are used, go ahead and remove them.